### PR TITLE
Create .travis.yml

### LINKED ISSUE #20: Set up CI for this template
CI should

* run the local version of the template (not 100% sure how to do this with `npm init` and local packages)
* verify that it builds afterwards (`npm run build`)

am I missing anything else it should do?

### PR DESCRIPTION
**Pull request by sendilkumarn**
fixes #20

### CODE REVIEW

**Comment by sendilkumarn** [comment]
@fitzgen Can you enable the Travis?

**Comment by fitzgen** [comment]
Enabled!

**Comment by fitzgen** [comment]
Empty commit to trigger the push?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    
+matrix:
+  include:
+    # tests pass on nightly
+    - rust: nightly
+      env: JOB=test-template
+      install:
+        - *INSTALL_NODE_VIA_NVM
+        - cargo install wasm-pack
+        - cargo install wasm-bindgen-cli
```
[the remark this answers is not in the document]
Comment (sendilkumarn) [reply .travis.yml]: @fitzgen It was not present when the PR was created. ¯\_(ツ)_/¯

**Comment by sendilkumarn** [comment]
Closing this in favor of #68

File: .travis.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (fitzgen) [reply .travis.yml]: Yep, it was the correct thing to do at the time, when other options did not exist.

**Pull request closed without merging**